include/muser.h

```diff
@@ -37,12 +37,16 @@
 #include <sys/uio.h>
 #include <unistd.h>
 
-#include "vfio_user.h"
 #include "pci.h"
-#include "caps/pm.h"
-#include "caps/px.h"
-#include "caps/msi.h"
-#include "caps/msix.h"
+#include "pci_caps/pm.h"
```

Comment:
Maybe we should put all PCI related stuff into a separate directory, e.g. `lib/pci/caps/pm.h`?

Reply:
Well these headers are public, so they'd be under include/pci/caps/ 
I can do that if you like along with include/pci/pci.h
Do you want lib/pci/muser_pci.c too?

Reply:
I'm going to merge this as is now, we can look at this again later if we like